server/core/hot_region_storage.go

```diff
@@ -162,7 +162,7 @@ func (h *HotRegionStorage) backgroundDelete() {
 	next := time.Date(now.Year(), now.Month(), now.Day(), defaultDeleteTime, 0, 0, 0, now.Location())
 	d := next.Sub(now)
 	if d < 0 {
-		d = d + 24*time.Hour
+		d += 24 * time.Hour
```

Comment:
some change fing in  weight_moving_average.go  eg. w.sum += n,should this need to uniform

Reply:
Would you like to leave a comment? I think this PR has already changed it which is mentioned in your example.